spec/dpl/providers/cloudfoundry_spec.rb

```diff
@@ -27,6 +27,10 @@
     it { should have_run './cf push -f other.yml' }
   end
 
+  describe 'given --strategy rolling' do
+    it { should have_run './cf push --strategy rolling' }
+  end
+
```

Comment:
@BanzaiMan do you know why it fails? Thanks!

Reply:
I think `--strategy` is reserved to vary provider implementation (see `pages` and `heroku`). We'll need a different name.

Reply:
I have no idea if the Cloudfoundry CLI will support more options in the future...

Reply:
Do you prefer `deployment_strategy` or `rolling`?